unity3d/Assets/Swrve/Editor/SwrveBuildComponent.cs

```diff
@@ -18,6 +19,34 @@ public class SwrveBuildComponent : SwrveCommonBuildComponent
     private static string IOSDemoBundleIdentifier = "com.swrve.demo";
     private static string AndroidPackageName = "com.example.gcm";
 
+	private const string POSTPROCESS_JSON = "Assets/Swrve/Editor/postprocess.json";
+	private const string TEMPLATE_CONTENT = "REPLACEME";
+	private const string PLOT_PUBLIC_TOKEN_KEY = "publicToken";
+	public const string PLOT_TOKEN_KEY = "PlotToken";
+    
+	private static Dictionary<string, object> postprocessJson = null;
```

Comment:
It's an editor object, so it's only alive during a single editor context.  Before / after pressing play, before / after building something.